src/ui/interaction/accordions/AvAccordionsGroup/AvAccordionsGroup.vue

```diff
@@ -0,0 +1,72 @@
+<script lang="ts" setup>
+import type AvAccordion from '@/ui/interaction/accordions/AvAccordion/AvAccordion.vue'
+import AvVIcon from '@/ui/base/AvVIcon/AvVIcon.vue'
+import { DsfrAccordion, DsfrAccordionsGroup } from '@gouvminint/vue-dsfr'
+import { useSlots, type VNode } from 'vue'
+
+/**
+ * Slots available in AvAccordions component.
+ * Used to inject accordions via `AvAccordion` components.
+ */
+defineSlots<{
+  /**
+   * Default slot for passing `AvAccordion` components.
+   */
+  default?: () => VNode<typeof AvAccordion>[]
+}>()
+
+const slots = useSlots()
+
+const accordionsItem = computed(() => slots.default?.() || [])
+const activeAccordion = ref<number>()
+
+defineExpose({ activeAccordion })
+</script>
+
+<template>
+  <DsfrAccordionsGroup v-model="activeAccordion">
+    <DsfrAccordion
+      v-for="(accordion, index) in accordionsItem"
+      :id="`accordion-${index}`"
+      :key="index"
```

Comment:
Pour eviter des problemes a11y
```suggestion
      :key="index"
      :aria-labelledby="`accordion-title-${index}`"
      :aria-describedby="`accordion-content-${index}`"
```

Reply:
Done!